grails-app/views/layouts/main_component.gsp

```diff
@@ -85,7 +84,8 @@
         contextPath: "${request.contextPath}",
         isAdmin: ${session.isAdmin ? session.isAdmin : false},
         isViewer: ${session.isViewer ? session.isViewer : false},
-        isBroad: ${session.isBroad ? session.isBroad : false}
+        clientId: "${grailsApplication.config.googleSignInClientId}"
+
       };
     </script>
```

Comment:
For a separate cleanup story, there are several functions here that look unused post-conversion:

    loadComments
    loadHistory
    toggleContinueMessage
    togglePropertyMessage

Reply:
US: https://broadinstitute.atlassian.net/browse/BTRX-793